examples/getting-started/external-context/app.js

```
@@ -16,6 +16,11 @@ export default class AppAnimationLoop extends MiniAnimationLoop {
   start(props) {
     const canvas = this._getCanvas(props);
 
+    this.isDemoSupported = Boolean(canvas.getContext('webgl'));
```

Under what circumstances would this be false?

---

IE

---

Like IE<11? I think it’s ok if we let the page crash.

---

No IE 11 requires "experimental-webgl". I prefer having IE not work than complicating the example, but leaving the check out causes the whole site to crash.

---

Just a thought - could these checks be moved into `AnimationLoop`? 

(Always looking for ways to minimize "clutter" in example code).

---

I'm thinking the way examples are handled needs a bit of a refactor, especially now that there are examples that can't use `AnimationLoop`.